egs2/wsj/asr1/conf/tuning/train_asr_transformer_maskctc.yaml

```diff
@@ -0,0 +1,65 @@
+batch_type: folded
```

Comment:
Can you add some remarks (comments) on which parts are different from the original training config?
Obviously decoder, right? Other parts?